astroplan/observer.py

```diff
@@ -745,8 +753,12 @@ def _calc_riseset(self, time, target, prev_next, rise_set, horizon,
         else:
             times = _generate_24hr_grid(time, -1, 0, N)
 
-        altaz = self.altaz(times, target, grid_times_targets=grid_times_targets)
-        altitudes = altaz.alt
+        if not self.trig_approx:
```

Comment:
(minor) generally it's recommended to do
```
if x:
    ...a...
else:
    ...b...
```
rather than
```

if not x:
    ...b...
else:
    ...a...
```